staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/types.go

```diff
@@ -50,6 +50,10 @@ type APIServiceSpec struct {
 	InsecureSkipTLSVerify bool
 	// CABundle is a PEM encoded CA bundle which will be used to validate an API server's serving certificate.
 	CABundle []byte
+	// Local means that the handling for the API groupversion is handled locally on this server.
+	// The call will simply delegate to the normal handler chain to be fulfilled.  This is mutually exclusive
+	// to Service and CABundle.
+	Local bool
```

Comment:
would it be cleaner to make `Service` optional? if you want to delegate to a service, include it, otherwise it gets handled locally?